domains_plugin.php

```diff
@@ -1218,6 +1218,20 @@ private function cronDomainTermChange()
 
         // Update the term for each service that is not on a 1 year pricing term
         foreach ($services as $service) {
+            // If the service has an open invoice, skip
+            $service_invoices = $this->Record->select(['invoices.*'])
+                ->from('services')
+                ->innerJoin('invoice_lines', 'invoice_lines.id', '=', 'services.id', false)
```

Comment:
This should use invoice_lines.service_id instead of invoice_lines.id

Also this branch should come from 1.7 instead of master